android-auto-app/src/main/java/com/mapbox/navigation/examples/androidauto/ReplayRouteTripSession.kt

```diff
@@ -9,7 +9,7 @@ import com.mapbox.navigation.core.replay.route.ReplayProgressObserver
 
 // TODO This will be deleted in favor of a public api
 @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
-internal class ReplayRouteTripSession : MapboxNavigationObserver {
+object ReplayRouteTripSession : MapboxNavigationObserver {
```

Comment:
Should you update current.txt?

Reply:
This is not SDK.

Reply:
Yeah we may move this to SDK at some point. It would be nice to have a high level controller for replay and a trip session, that is used by dropin and androidauto.

But at this point the responsibility is on the developer downstream